CHANGES.txt

```diff
@@ -5,6 +5,12 @@ CHANGES for Crate ReadTheDocs Theme
 Unreleased
 ----------
 
+2019/05/08 0.5.75
```

Comment:
The new version should only be added when preparing the release
https://github.com/crate/crate-docs-theme/blob/master/DEVELOP.rst#preparing-a-release